inst/app/server.R

```diff
@@ -814,6 +814,30 @@ ORDER BY gri.sample_id;
 
   # Samples Check-in --------------------------------------------------
 
+  output$check_in_preview <- renderTable({
+    # This table shows a preview of the loaded file
+    if(is.null(input$check_in_samples_file$datapath)){
```

Comment:
this is a good use case for `need` from shiny: https://shiny.posit.co/r/reference/shiny/0.11/validate.html it will allow you to check for the file and if not available you can show a message